node/src/components/consensus/era_supervisor.rs

```diff
@@ -85,10 +86,10 @@ pub struct EraSupervisor<I> {
     ///
     /// This map always contains exactly `2 * bonded_eras + 1` entries, with the last one being the
     /// current one.
-    open_eras: HashMap<EraId, Era<I>>,
+    pub(super) open_eras: HashMap<EraId, Era<I>>,
     secret_signing_key: Arc<SecretKey>,
     public_signing_key: PublicKey,
-    current_era: EraId,
+    pub(super) current_era: EraId,
```

Comment:
I'm not sure how @afck feels about it but I would prefer to stick to the approach where we have methods to access this rather than making fields accessible directly.

Reply:
My bad, that was an oversight. I initially had all these reverted to accessor methods once I saw the pattern, but these two slipped through. I'll fix it right away.

Reply:
Also, shoutout to rust-analyzer!

```rust
    /// Get a reference to the era supervisor's open eras.
    pub fn open_eras(&self) -> &HashMap<EraId, Era<I>> {
        &self.open_eras
    }
```

The above was fully autogenerated, including the comment. :exploding_head: